src/storage/memory_cache/tests.rs

```diff
@@ -152,203 +182,225 @@ async fn prepare_data_for_evict() -> (Arc<MemoryStorage>, Arc<MemoryCacheType>)
     (backend, cache)
 }
 
-#[tokio::test]
-async fn test_evict() {
-    let (backend, cache) = prepare_data_for_evict().await;
-
-    // LRU in cache: (0, 0) -> (0, 1) -> (0, 2) -> (0, 3)
-    // Insert a block, and (0, 0) will be evicted
-    assert!(!backend.contains(0, 0));
-    let block = Block::from_slice(BLOCK_SIZE_IN_BYTES, BLOCK_CONTENT);
-    cache.store(1, 0, block).await.unwrap();
-    let loaded = backend.load(0, 0).await.unwrap().unwrap();
-    assert_eq!(loaded.as_slice(), BLOCK_CONTENT);
-}
+#[rstest]
+fn test_evict() {
+    shared_runtime_test(async {
+        let (backend, cache) = prepare_data_for_evict().await;
 
-#[tokio::test]
-async fn test_touch_by_load() {
-    let (backend, cache) = prepare_data_for_evict().await;
-
-    // LRU in cache: (0, 0) -> (0, 1) -> (0, 2) -> (0, 3)
-    // Touch (0, 0) by loading
-    let _: Option<Block> = cache.load(0, 0).await.unwrap();
-
-    // LRU in cache: (0, 1) -> (0, 2) -> (0, 3) -> (0, 0)
-    // Insert a block, and (0, 1) will be evicted
-    assert!(!backend.contains(0, 1));
-    let block = Block::from_slice(BLOCK_SIZE_IN_BYTES, BLOCK_CONTENT);
-    cache.store(1, 0, block).await.unwrap();
-    let loaded = backend.load(0, 1).await.unwrap().unwrap();
-    assert_eq!(loaded.as_slice(), BLOCK_CONTENT);
+        // LRU in cache: (0, 0) -> (0, 1) -> (0, 2) -> (0, 3)
+        // Insert a block, and (0, 0) will be evicted
+        assert!(!backend.contains(0, 0));
+        let block = Block::from_slice(BLOCK_SIZE_IN_BYTES, BLOCK_CONTENT);
+        cache.store(1, 0, block).await.unwrap();
+        let loaded = backend.load(0, 0).await.unwrap().unwrap();
+        assert_eq!(loaded.as_slice(), BLOCK_CONTENT);
+    });
 }
 
-#[tokio::test]
-async fn test_touch_by_store() {
-    let (backend, cache) = prepare_data_for_evict().await;
-
-    // LRU in cache: (0, 0) -> (0, 1) -> (0, 2) -> (0, 3)
-    // Touch (0, 0) by storing
-    let block = Block::from_slice(BLOCK_SIZE_IN_BYTES, BLOCK_CONTENT);
-    cache.store(0, 0, block).await.unwrap();
-
-    // LRU in cache: (0, 1) -> (0, 2) -> (0, 3) -> (0, 0)
-    // Insert a block, and (0, 1) will be evicted
-    assert!(!backend.contains(0, 1));
-    let block = Block::from_slice(BLOCK_SIZE_IN_BYTES, BLOCK_CONTENT);
-    cache.store(1, 0, block).await.unwrap();
-    let loaded = backend.load(0, 1).await.unwrap().unwrap();
-    assert_eq!(loaded.as_slice(), BLOCK_CONTENT);
+#[rstest]
+fn test_touch_by_load() {
+    shared_runtime_test(async {
```

Comment:
task manager have a runtime, do not need shared_runtime_test anymore

Reply:
OK, I have revert this commit, it is not necessary here by shared runtime.